CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Add the sys.summits table.
```

Comment:
Please use past tense: Added, and maybe add some more description about this table